test/core_ocaf_unittest.py

```diff
@@ -119,6 +119,67 @@ def test_read_step_file(self) -> None:
         a_shape = shape_tool.GetShape(label_shp)
         self.assertFalse(a_shape.IsNull())
 
+    def test_read_write_step_string(self) -> None:
+        """Writes a step file as a string and reads it back"""
```

Comment:
I thought that doing a round trip would be the simplest way to ensure both writing and reading have assertions applied to them. I noticed that `test_write_step_file` may not have any assertions being applied